src/utils/_forms.scss

```diff
@@ -29,3 +29,93 @@
     display: none;
   }
 }
+
+//.sui-Form {
+//  @include form-field(input);
+//}
+
+//<div class="sui-AtomInput sui-AtomInput-success">
```

Comment:
Classnames are not following SUIT CSS as we're using in Schibsted. https://suitcss.github.io/

`sui-AtomInput sui-AtomInput--success`
`sui-AtomInput-inputWrapper`
`sui-AtomInput--medium`